packages/dso-toolkit/src/components/tabs/tabs.mixins.scss

```diff
@@ -0,0 +1,189 @@
+@use "../anchor";
```

Comment:
Ik zie het harde werk wat hier is gedaan, maar ik vraag mij af of we het goede doen.

De huidige HTML/CSS implementatie is afgeleid van Bootstrap. Die is gebasseerd op het concept "ge hebt nav, en van nav kan ge tabs maken".

En daar komt die suffe `nav` selector ook bij om de hoek kijken. En eigenlijk wil ik daar helemaal vanaf stappen.

Ik stel het volgende voor:
* We reverten de CSS implementatie terug naar wat ie was, dus geen centrale styling met `.mixins.scss`
* De stylesheets behorend bij de Web Components (`dso-tabs` en `dso-tab`) zijn zelfstandige stylesheets
* De markup in `<dso-tabs>` en `<dso-tab>` kan daarmee alle vormen (elementen en classes etc) aannemen die nodig is.
* Vaste waardes zoals afstanden en kleuren kunnen we natuurlijk centraliseren naar variables.